mutatest/run.py

```diff
@@ -281,91 +283,80 @@ def trial_output_check_break(
     Returns:
         Bool flag for whether or not to break the outer operations loop.
     """
-    if trial_results.status == "SURVIVED":
-        LOGGER.info(
-            "%s",
-            colorize_output(
-                (
-                    f"Surviving mutation at "
-                    f"{sample_src}: ({sample_idx.lineno}, {sample_idx.col_offset})"
-                ),
-                "red",
-            ),
-        )
-        if config.break_on_survival:
-            LOGGER.info(
-                "%s",
-                colorize_output(
-                    "Break on survival: stopping further mutations at location.", "red"
-                ),
-            )
-            return True
 
-    if trial_results.status == "DETECTED":
-        LOGGER.info(
-            "%s",
-            colorize_output(
-                (
-                    f"Detected mutation at "
-                    f"{sample_src}: ({sample_idx.lineno}, {sample_idx.col_offset})"
-                ),
-                "green",
-            ),
-        )
-        if config.break_on_detected:
+    @dataclass
```

Comment:
This refactor was done in order to reduce the complexity of the function (partly to pass linting checks)

Reply:
Yup, this look good to me, and an overdue change.